src/components/inputs/toggle/toggle.component.tsx

```diff
@@ -57,16 +56,17 @@ const Toggle: React.FC<FormFieldProps> = ({ question, onChange, handler, previou
     !question.isHidden && (
       <div className={styles.boldedLabel}>
         <ToggleInput
-          labelText={t(question.label)}
-          className={styles.boldedLabel}
-          id={question.id}
-          labelA={question.questionOptions.toggleOptions.labelFalse}
-          labelB={question.questionOptions.toggleOptions.labelTrue}
-          onToggle={handleChange}
-          toggled={!!field.value}
-          disabled={question.isDisabled}
-          readOnly={question.readonly}
-        />
+  labelText={<QuestionLabelContainer question={question} />}
```

Comment:
This indenting is off, could you please indent props a tab inside the component like it was before